README.md

```diff
@@ -1,2 +1,47 @@
 # Capstone-Project-m1
-Module-1 Capstone Project
+## Description
+- Module-1 Capstone Project is the last project of the module 1 leraning and coding activities at Microverse.
+## Learning Objectives
+- Use semantic HTML tags.
+- Apply best practices in HTML code.
+- use CSS selectors correctly.
+- Use CSS box model.
+- Use flex box to place elements in the page.
+- Demostrate ability to create UIs adaptable to different screen sizes using media queries.
+- Use GitHub Pages to deploy web pages.
+- Apply JavaScript best practices and language style giudes in code.
+- Use JavaScript to manipulate DOM elements.
+- Use JavaScript events.
+- Use objects to store and access data.
+- communicate technical concepts to other technical people.
+## Built with
+- HTML, CSS, and JAVASCRIPT,
+- Visual studio code, Git and GitHub, GitHub Desktop and Git Bash.
+## Live Demo (if available)
+
+[Live Demo Link](https://lugard1.github.io/Capstone-Project-m1/)
+
+## Live Presentation video link of my Capstone-Project-M1
+- https://www.loom.com/share/fdcc1e4c655146c78e4172750545b09a
```

Comment:
Good job creating the video as suggested by the previous reviewer however, I suggest you make a video that is not more than 5 minutes as per the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md)
<img width="1673" alt="Screenshot 2022-10-29 at 17 01 28" src="https://user-images.githubusercontent.com/48546992/198841882-0a9552a2-887e-4bbf-872c-ee220fbdf620.png">

Your current video is too long at 9 minutes

Reply:
- [ ] This issue is well resolved :heavy_check_mark: